recipes/default.rb

```diff
@@ -11,6 +11,11 @@
 end.run_action(:upgrade)
 
 chef_gem 'ffi-libarchive' do
-  version '~> 0.2.0'
+  version '= 0.2.0'
+  compile_time true
+end
+
+chef_gem 'ffi' do
```

Comment:
Should we put `ffi` before `ffi-libarchive` here? I'm not 100% sure, but the reason I think it might be necessary is that `ffi-libarchive` will attempt to install `ffi` (whatever version it wants) before we hit your addition with the pinned version.